2-mandatory/4-people-i-know.js

```diff
@@ -402,6 +419,16 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+friends.forEach((friend) => {
+  const colleagues = friend.colleagues;
+  for (const colleague in colleagues) {
+    const isMultiTasking =
+      colleagues[colleague].skills.includes("Multi-tasking");
+    if (isMultiTasking) {
+      colleaguesWhoCanMultitask.push(colleagues[colleague].name);
+    }
```

Comment:
Make sure before using FOR IN, check whether the object has that object or not. Usually, VScode will suggest to you. If you don't check the loop won't work.

Reply:
Thank you Ferzad.